mmdb/init.lua

```diff
@@ -320,10 +333,10 @@ getters[32] = {
 function geodb_methods:search(bits, node)
 	node = node or 0
 	local seen = { [node] = true }
-	for _, direction in ipairs(bits) do
+	for i = 1, #bits do
```

Comment:
In theory it more faster way to do iteration. So I just prefer this one.
I saw this in several places. e.g https://springrts.com/wiki/Lua_Performance#TEST_9:_for-loops.